app/src/main/res/layout/layout_contribution.xml

```diff
@@ -92,16 +113,6 @@
                 android:padding="8dp"
                 />
 
-            <ImageButton
```

Comment:
Instead of removing this button, please just add a lisstle space between this button and cancel button.

Reply:
I thought the retry button will provide a better UI, on the left side of the title. Should I remove that entirely,  keep them on the right side and only add some space between the buttons?

Reply:
Yes just a bit more space between the two would be the best, I think :-)